lib/core/test/unit/Cardano/Wallet/ApiSpec.hs

```diff
@@ -591,3 +605,4 @@ titleize proxy req = unwords
     ]
   where
     proxyStr = "(" <> show (typeRep proxy) <> ")"
+
```

Comment:
Request: remove extraneous line from end of file.

Perhaps your editor is inserting an extra CR at the end of the file? :space_invader: